Modules/Scripted/Endoscopy/Endoscopy.py

```diff
@@ -63,7 +65,7 @@ def setup(self):
 
     # Path collapsible button
     pathCollapsibleButton = ctk.ctkCollapsibleButton()
-    pathCollapsibleButton.text = "Path"
+    pathCollapsibleButton.text = tr("EndoscopyWidget", "Path")
```

Comment:
Would be really nice if we could have a decorator to supply the translation context just to make things less verbose.